Documentation/compatibility/winforms-accessibility-changes-472.md

```diff
@@ -0,0 +1,109 @@
+- <xref:System.Windows.Forms.DataGridViewLinkButton> will by default use OS-defined colors in High Contrast mode for the <xref:System.Windows.Forms.DataGridViewLinkButton.LinkColordisplayProperty=nameWithType> property.
+
+NOTE: Windows 10 has changed values for some high contrast system colors. Windows Forms Framework is based on the Win32 framework. For the best experience, run on the latest version of Windows and opt in to the latest OS changes by adding an app.manifest file in a test application and uncommenting the following code:
+```
+    <!-- Windows 10 -->
+    <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
+```
+
+__Improved Narrator support__
+- Narrator now announces the value of the <xref:System.Windows.Forms.Keys.ShortcutKeys?displayProperty=nameWithType> property when announcing the text of a <xref:System.Windows.Forms.ToolStripMenuItem>. 
+- Narrator now indicates when a <xref:System.Windows.Forms.ToolStripMenuItem> has its <xref:System.Windows.Forms.Control.Enabled> property to 'false'.
+- Narrator now gives feedback on the state of a check box when the <xref:System.Windows.Forms.ListView.CheckBoxes?displayProperty=fullName> property is set to 'true'.
+- Narrator Scan Mode focus order is now consistent with the visual order of the controls on the ClickOnce download dialog window.
+
+__Improved DataGridView Accessibility support__
+- Rows in a <xref:System.Windows.Forms.DataGridView> can now be sorted using the keyboard. Now a user can use the F3 key in order to sort by the current column.
+- When the <xref:System.Windows.Forms.DataGridView.SelectionMode?displayProperty=nameWithType> is set to <xref:System.Windows.Forms.DataGridViewSelectionMode.FullRowSelect?displayProperty=nameWithType> the column header will change color to indicate the current column as the user tabs through the cells in the current row.
+- The <xref::System.Windows.Forms.DataGridViewCell.DataGridViewCellAccessibleObject.Parent?displayProperty=nameWithType> property of a <xref:System.Windows.Forms.DataGridViewCell.DataGridViewCellAccessibleObject> now returns the correct parent control.
+
+__Improved Visual cues__
+- <xref:System.Windows.Forms.RadioButton> and <xref:System.Windows.Forms.CheckBox> controls with an empty Text:<xref:System.Windows.Forms.ButtonBase.Text> property will now display a focus indicator when they recieve focus.
+
+__Improved Property Grid Support__
+- The <xref:System.Windows.Forms.PropertyGrid> control child elements now return a 'true' for the  <xref:System.Windows.Automation.ValuePattern.IsReadOnlyProperty> property only when a PropertyGrid element is enabled.
+- The <xref:System.Windows.Forms.PropertyGrid> control child elements now return a 'false' for the <xref:System.Windows.Automation.AutomationElement.IsEnabledProperty>property only when a PropertyGrid element can be changed by the user.

```

Comment:
- 'false' should be fenced (\`false\`)
- space between IsEnabledProperty and "property"

Reply:
done